documentation/how-to/prod-checklist.md

```diff
@@ -0,0 +1,81 @@
+---
+title: {Production Readiness Checklist}
+---
+
+This guide gives
+{{ glossary_tooltip term_id="api-provider" text="API Providers" }} a quick
+reference of actions they need to complete before publishing their Gateway
+Services to the {{ glossary_tooltip term_id="api-directory" }}.
+
+## First time activities
+
+- Learn about and understand the benefits of the
+  {{ glossary_tooltip term_id="api-services-portal" }} by reviewing our [plain language](https://www2.gov.bc.ca/gov/content/data/finding-and-sharing/api-management)
+  and [technical documentation](https://developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/).
+  Reach out to the APS team by opening a [support ticket](https://dpdd.atlassian.net/servicedesk/customer/portal/1/group/2)
+  or on our [Rocket.Chat](https://docs.developer.gov.bc.ca/join-bc-rocket-chat/)
+  channel (#aps-ops) if you can’t find the information you’re looking for.
+
+- Have access to a production-ready API. This could be a public or private
```

Comment:
These points could be made easier to read at a glance by:
- using a short introductory sentence to capture the point
- making the first sentence bold, and/or 
- adding a line break moving the additional details in following sentences below the intro sentence, in the same bullet point